### PR TITLE
NT-0044 @testing-library/react

### PR DESCRIPTION
**Pull request by Foukz**

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -16,7 +16,9 @@
     "@material-ui/icons": "^4.0.0",
     "axios": "^0.19.0",
     "formik": "^1.5.7",
+    "jest-dom": "^3.5.0",
     "jwt-decode": "^2.2.0",
+    "prettier": "^1.18.2",
```
Comment (mras2303) [inline client/package.json]: Are you sure that prettier should be here?

1. This library is using only for developers, so it should move to devDependencies
2. It should be moved to root directory

File: client/src/containers/SignUp/SignUpForm/SignUpForm.tsx [hunk 2]
```diff
@@ -111,6 +114,7 @@ const SignUp: React.FC<Props> = ({ onSubmit }) => (
           label="Password*"
           outllined
           component={TextField}
+          id="password"
```
Comment (mras2303) [inline client/src/containers/SignUp/SignUpForm/SignUpForm.tsx]: Why do we need id property in these fields?

File: client/package.json (hunk 1 above)
Comment (Foukz) [reply client/package.json]: Good catch with devDependencies, thanks :smile: 
Prettier is needed for creating snapshots and without it jest throws an error

File: client/src/containers/SignUp/SignUpForm/SignUpForm.test.tsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  const handleSubmit = jest.fn();
+  const { getByText, queryAllByText } = render(
+    <SingUp onSubmit={handleSubmit} />
+  );
+
+  fireEvent.click(getByText(/sign up/i).parentElement!);
+  await wait();
+  expect(queryAllByText(/ *[A-Z]* required/i)).toMatchInlineSnapshot(`
+    Array [
+      <p
```
Comment (mras2303) [inline client/src/containers/SignUp/SignUpForm/SignUpForm.test.tsx]: ouch this formatting looks horrible. Could you rewrite it to below example? Is this possible? And if it is possible, could you move that inline styling to css file?

`<p style="margin-bottom: 5px;"> - First name is required</p>`

File: client/package.json (hunk 1 above)
Comment (mras2303) [reply client/package.json]: he? Could you provide us this error message?

File: client/src/containers/SignUp/SignUpForm/SignUpForm.tsx (hunk 2 above)
Comment (Foukz) [reply client/src/containers/SignUp/SignUpForm/SignUpForm.tsx]: 1. This is material-ui thing. id property is getting transformed to 
```
<label for={valueInId} ></label>
<input id={valueInId} />
```
I think ` for` in labels is always good for semantic reasons
2. Thanks to that during tests we can in easy way query inputs by `getByLabeLText`

File: client/package.json (hunk 1 above)
Comment (Foukz) [reply client/package.json]: `Cannot find module 'prettier' from 'setup_jest_globals.js'`

**Pull request closed without merging**